projects/Mallard/ios/Mallard/Ophan/Ophan.swift

```diff
@@ -44,13 +44,15 @@ class Ophan: NSObject {
         $0.withMemoryRebound(to: CChar.self, capacity: 1) { String(validatingUTF8: $0) }
     }
     let deviceName = modelCode ?? "Unrecognised model"
+    let deviceClass = UIDevice.current.userInterfaceIdiom == .pad ? Multiplatform_ophanDeviceClass.tablet : Multiplatform_ophanDeviceClass.phone
```

Comment:
`.pad`? is an odd construct - what is it?

Reply:
In swift if something is expecting a type in a specific position (here it's probably something like `UserInterfaceIdiom` for the comparison operator) then, if then if there is a static member on that type class that yields a value of that type you can omit the class name.